spec/classes/init_spec.rb

```diff
@@ -430,6 +430,13 @@
             })
           end
 
+          it do
+            should contain_exec('pam-config -a --mkhomedir-umask=0022').with({
+              :path   => '/bin:/usr/bin:/sbin:/usr/sbin',
+              :unless => '/usr/sbin/pam-config -q --mkhomedir | grep umask=0022',
+            })
+          end
+
```

Comment:
Could you please add a test that shows that if you specify `pam_mkhomedir_umask` to a non-default value (not `'0022'`) that the exec will have that and that the content of the file resource will also reflect the change.